installers/go-server/win/config/wrapper-server.conf

```diff
@@ -54,19 +54,18 @@ set.JAVA_HOME=%GO_SERVER_JAVA_HOME%
 
 wrapper.java.additional.1=-Xms512m
 wrapper.java.additional.2=-Xmx1024m
-wrapper.java.additional.3=-XX:PermSize=128m
-wrapper.java.additional.4=-XX:MaxMetaspaceSize=256m
-wrapper.java.additional.5=-Duser.language=en
-wrapper.java.additional.6=-Duser.country=US
-wrapper.java.additional.7="-Dcruise.config.dir=%CRUISE_SERVER_DIR%\config"
-wrapper.java.additional.8="-Dcruise.config.file=%CRUISE_SERVER_DIR%\config\cruise-config.xml"
-wrapper.java.additional.9=-DreservedForFuture.1
-wrapper.java.additional.10=-DreservedForFuture.2
-wrapper.java.additional.11="-DJAVA_SYS_MON_TEMP_DIR=%CRUISE_SERVER_DIR%\tmp"
-wrapper.java.additional.12=%JVM_DEBUG%
-wrapper.java.additional.13=%GC_LOG%
-wrapper.java.additional.14=-DreservedForFuture.4
-wrapper.java.additional.15=-DreservedForFuture.5
+wrapper.java.additional.3=-XX:MaxMetaspaceSize=256m
+wrapper.java.additional.4=-Duser.language=en
+wrapper.java.additional.5=-Duser.country=US
+wrapper.java.additional.6="-Dcruise.config.dir=%CRUISE_SERVER_DIR%\config"
+wrapper.java.additional.7="-Dcruise.config.file=%CRUISE_SERVER_DIR%\config\cruise-config.xml"
+wrapper.java.additional.8=-DreservedForFuture.1
+wrapper.java.additional.9=-DreservedForFuture.2
+wrapper.java.additional.10="-DJAVA_SYS_MON_TEMP_DIR=%CRUISE_SERVER_DIR%\tmp"
+wrapper.java.additional.11=%JVM_DEBUG%
+wrapper.java.additional.12=%GC_LOG%
+wrapper.java.additional.13=-DreservedForFuture.4
+wrapper.java.additional.14=-DreservedForFuture.5
```

Comment:
These variables are expected to be overridden in another file. As such, changing the indexes of properties will likely cause the orverrides to break.

For e.g. A user may have specified `-wrapper.java.additional.5=-Duser.language=foo`. However this will now override `user.country` instead.

It is recommended to set a dummy value for a property that is removed `-DreservedForFuture.X`.

In fact, if it's not too much to ask, could you put this comment in the file, for the next person who edits it :)